api/config/appstudio.redhat.com_enterprisecontractpolicies.yaml

```
@@ -83,6 +83,9 @@ spec:
                 rekorUrl:
                   description: URL of the Rekor instance. Empty string disables Rekor integration
                   type: string
+                ruleData:
+                  description: A json or yaml string containing rule data that will be visible to policy rules
```

It's no longer a json or yaml string, right? Maybe:

```suggestion
                  description: Arbitrary rule data that will be visible to policy rules
```